### PR TITLE
 bitwise_and

### LINKED ISSUE #5857: bitwise_and

### PR DESCRIPTION
**Pull request by AlphaPro06**
Close #5857

### CODE REVIEW

**Comment by AlphaPro06** [comment]
Close #5857

**Comment by Nightcrab** [comment]
Please correct the indentation in `ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py`

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 1]
```diff
@@ -1133,3 +1133,33 @@ def test_jax_numpy_arcsinh(
         fn_tree="numpy.arcsinh",
         x=x[0],
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("bool"),
```
Comment (Nightcrab) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1141]: The test should also provide cases for integral inputs, not just booleans.

**Review by Nightcrab — CHANGES_REQUESTED** [review]

**Comment by AlphaPro06** [comment]
Hello @Nightcrab, can you please point me where do you think the indentation is incorrect?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py (hunk 1 above)
Comment (AlphaPro06) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1141]: Can you please elaborate on what do you mean for integral inputs?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 2]
```diff
@@ -48,10 +48,10 @@ def test_jax_numpy_abs(
...
 )
 def test_jax_numpy_absolute(
-    dtype_and_x,
-    as_variable,
-    num_positional_args,
-    native_array,
+        dtype_and_x,
+        as_variable,
+        num_positional_args,
+        native_array,
```
Comment (Nightcrab) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:54]: These indentations were changed from 4 spaces to 8; 4 space indentation should be used for the test function arguments.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py (hunk 1 above)
Comment (Nightcrab) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1141]: integer arrays.

**Comment by AlphaPro06** [comment]
@Nightcrab I think I have made the requested changes you've asked me to make. Please let me know if there is anything else I need to add!

**Comment by Nightcrab** [comment]
What's the ``arrs_and_scalars_for_bitwise_and`` function for? It doesn't seem to be referenced anywhere.

**Comment by AlphaPro06** [comment]
Oh never mind that, I was trying something different and forgot to remove that.

**Comment by AlphaPro06** [comment]
I just removed the extra function you mentioned

**Comment by Nightcrab** [comment]
Looks alright apart from some over-indents. It seems like Jax tests are not working properly at the moment so I'll have to do the review/merge once they are.

**Comment by Nightcrab** [comment]
Merged.

**Comment by AlphaPro06** [comment]
Thank you!

**Pull request merged**